apps/api/src/database/migrations/2692870736644-migration.ts

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Is there any specific reason to disable Prettier in this file? Can remove this if not.

Reply:
Will remove this